docs/src/1_Mechanics/01_PlayerGuide_Full.md

```diff
@@ -545,35 +548,33 @@ one level of *Fatigue*. Reshuffle your deck and draw all Checks with the
 Lower Hand. Each time you need to reshuffle your deck in this way adds
 another level of Fatigue. If you were to suffer a fourth level of Fatigue,
 you are Knocked Out instead. This signifies your character running out of
-Stamina during a long day. 
+Stamina during a long day.
 
-### Discard Mechanics
+### Discard Powers
 
-Each Dealer may have an additional Burn Mechanic. Dealers may discard cards as
+Each Dealer may have one Discard Power. Dealers may discard cards as
 a Minor Action for additional benefits in combat starting at Level 2. For
 every 2 cards discarded, up to a limit of 4, an Upper/Lower Hand +1 is
 granted described below. These effects do not stack; only one may be applied
-to a character at a time.
+to a character at a time. Cards are discarded without being turned over, and cannot
+yeild Fate Cards.
```

Comment:
Did we decide that discarded Aces can't be used as Fate Cards? I thought by the 'open discard pile' philosophy we'd decided you can see your cards at all times - I'm OK with Aces that are discarded in this manner being unable to be reserved as Fate Cards (makes the mechanic more risky so less likely to be over-used) but I think players should still see the cards they discard to avoid the discard pile becoming something they can't see or interact with over time.

Reply:
I'll remove 'without being turned over'